apps/src/sites/studio/pages/congrats/index.js

```diff
@@ -0,0 +1,17 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Notification from '@cdo/apps/templates/Notification';
+
+$(document).ready(function () {
```

Comment:
Not needed if you take out `content_for(:head) do ...` below.

Reply:
How so?